lib/mix/tasks/tower_bugsnug.install.ex

```diff
@@ -0,0 +1,188 @@
+if Code.ensure_loaded?(Igniter) && Code.ensure_loaded?(Tower.Igniter) do
+  defmodule Mix.Tasks.TowerBugsnug.Install do
+    @example "mix igniter.install tower_bugsnug"
+
+    @shortdoc "Installs TowerBugsnug. Invoke with `mix igniter.install tower_bugsnug`"
+    @moduledoc """
+    #{@shortdoc}
+
+    ## Example
+
+    ```bash
+    #{@example}
+    ```
+    """
+
+    use Igniter.Mix.Task
+
+    import Tower.Igniter
+
+    @default_runtime_config """
+    import Config
+
+    if config_env() == :prod do
+      config :tower_bugsnug, api_key: System.get_env("BUGSNUG_API_KEY")
+    end
+    """
+
+    @prod_config_patterns [
+      """
+      if config_env() == :prod do
+        __cursor__()
+      end
+      """,
+      """
+      if :prod == config_env() do
+        __cursor__()
+      end
+      """
+    ]
+
+    @bugsnug_config_code """
+    config :tower_bugsnug, api_key: System.get_env("BUGSNUG_API_KEY")
+    """
+
+    @impl Igniter.Mix.Task
+    def info(_argv, _composing_task) do
+      %Igniter.Mix.Task.Info{
+        group: :tower,
+        adds_deps: [],
+        installs: [],
+        example: @example,
+        only: nil,
+        positional: [],
+        composes: [],
+        schema: [],
+        defaults: [],
+        aliases: [],
+        required: []
+      }
+    end
+
+    @impl Igniter.Mix.Task
+    def igniter(igniter) do
+      igniter
+      |> add_reporter_to_config(TowerBugsnug)
```

Comment:
It's `Bugsnag`, not `Bugsnug`.

Reply:
fixed https://github.com/mimiquate/tower_bugsnag/pull/60/commits/18cdf3bcb1a589f3706cb6e5e5f2898edc85ae8e